app/uk/gov/hmrc/cdsreimbursementclaimfrontend/config/ViewConfig.scala

```diff
@@ -47,18 +48,23 @@ class ViewConfig @Inject() (config: Configuration, servicesConfig: ServicesConfi
     servicesConfig.getDuration("gg.countdown").toSeconds
 
   val ggKeepAliveUrl: String =
-    "/claim-for-reimbursement-of-import-duties" + routes.StartController.keepAlive().url
+    "/claim-for-reimbursement-of-import-duties" + baseRoutes.StartController.keepAlive().url
 
   val ggTimedOutUrl: String =
-    signOutUrl + "?continue=/claim-for-reimbursement-of-import-duties" + routes.StartController
+    signOutUrl + "?continue=/claim-for-reimbursement-of-import-duties" + baseRoutes.StartController
       .timedOut()
       .url
 
   val ggSignOut: String =
-    signOutUrl + "?continue=/claim-for-reimbursement-of-import-duties" + routes.StartController
+    signOutUrl + "?continue=/claim-for-reimbursement-of-import-duties" + baseRoutes.StartController
       .start()
       .url
 
+  def claimTimedOutUrl(journeyBindable: JourneyBindable): String =
+    "https://www.tax.service.gov.uk/claim-for-reimbursement-of-import-duties" + claimsRoutes.CheckClaimantDetailsController
```

Comment:
Why this URL is hardcoded - https://www.tax.service.gov.uk/claim-for-reimbursement-of-import-duties?

Reply:
Because that's the only way I could get the page to re-direct